config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+
+   get 'users/:id/posts', to: 'posts#index'
+   get 'users/:id/posts/:id', to: 'posts#show'
+
+   get 'users/', to: 'users#index'
+   get 'users/:id', to: 'users#show'
+  
```

Comment:
- [ ] Rails has methods that help us define these routes. Let's use those instead and keep our code simple. If we leave it as below, then we would not be able to tell which id is being used when we call `params[:id]`
  
  ```
  get 'users/:id/posts/:id', to: 'posts#show'
  ```
  I would recommend you change your routes to the code below. This allows us to differentiate between `user_id` and post `id`
  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```